src/plextrac

```diff
@@ -237,7 +237,13 @@ function mod_install() {
   title "Installing PlexTrac Instance"
   requires_user_plextrac
   mod_configure
-  mod_check
+  info "Checking Docker Compose Config"
+  compose_client config -q && info "Config check passed"
+  pending=`composeConfigNeedsUpdated || true`
+  if [ "$pending" != "" ]; then
+      error "Pending Changes:"
+      msg "    %s\n" "$pending"
+  fi
```

Comment:
```suggestion
```

this logic can be omitted since all we want to check here is that the Compose syntax is valid